src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -1237,16 +1247,81 @@ impl A320HydraulicBrakingLogic {
 
 impl SimulationElement for A320HydraulicBrakingLogic {
     fn read(&mut self, state: &mut SimulatorReader) {
-        self.parking_brake_demand = state.read_bool("BRAKE PARKING INDICATOR");
+        self.parking_brake_demand = state.read_bool("BRAKE PARKING POSITION");
         self.weight_on_wheels = state.read_bool("SIM ON GROUND");
         self.is_gear_lever_down = state.read_bool("GEAR HANDLE POSITION");
         self.anti_skid_activated = state.read_bool("ANTISKID BRAKES ACTIVE");
-        self.left_brake_pilot_input = state.read_f64("BRAKE LEFT POSITION") / 100.0;
-        self.right_brake_pilot_input = state.read_f64("BRAKE RIGHT POSITION") / 100.0;
+        self.left_brake_pilot_input = state.read_f64("BRAKE LEFT POSITION");
+        self.right_brake_pilot_input = state.read_f64("BRAKE RIGHT POSITION");
         self.autobrakes_setting = state.read_f64("AUTOBRAKES SETTING").floor() as u8;
     }
 }
 
+struct A320BrakingForceSimulationOutput {
```

Comment:
**suggestion**: We are in the simulation and that we output information is to be expected. Consider renaming this and variables refering to this to something like `A320BrakingForce`.